Python/di_sensors/dexter_i2c.py

```diff
@@ -11,6 +11,19 @@
 
 import time
 
+#RPI_1_Module = "pigpio"
```

Comment:
@CleoQc can you explain why we're doing this?  This seems like a confusing way to change around the library we're using.

Reply:
Commenting here would be really helpful.

Reply:
That's @mattallen37 ' code, as is. I'll add a couple of comments.